### PR TITLE
Credit page added

### PR DESCRIPTION
**Pull request by SilentFlame**
Page has the names and links regarding the contributors.
Issue #58 part1 resolved.

### CODE REVIEW

File: javascript/links.js [hunk 1]
```diff
@@ -13,16 +12,16 @@ function android()
...
-	if(document.getElementById('x4').innerHTML.indexOf('i')!=1)
-	{
-		window.location="https://github.com/fossasia/flappy-svg/blob/master/FlappySVG_Android/app-release.apk?raw=true";
 	}
-	else
-	{
+}
+// function contributors()
+// {
+// 	if(document.getElementById('x4').innerHTML.indexOf('i')!=1)
```
Comment (niccokunzmann) [inline javascript/links.js]: Why is the one function removed and the other function is commented out?

File: credits.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			<a href="links.html"><img src="back.png" style="position:absolute;width:50px;margin-top:10px;" /></a>
+			<!--<div id="linkbox" > -->
+			<br><br>
+			<h1>Contributors</h1>
+			<h3>These people have contributed to this game: </h3>
+			<ul>
+				<li><a href="http://jugendprogrammiert.weebly.com/">Nicco Kunzmann</a></li>
+				<li><a href="https://github.com/PohSayKeong">Poh Say Keong</a></li>
+				<li><a href="https://github.com/shashank-sharma">Shashank Sharma</a></li>
+				<li><a href="(https://github.com/anniezhou301">Annie Zhou</a></li>
```
Comment (niccokunzmann) [inline credits.html]: ( is wrong

File: credits.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+				</ul>
+
+		<!--</div> -->
+		</div>
+		<div class="col-md-4" style="height:100%;background-color:#000">
+		</div>
+	</div>
+</div>
+
+</body>
```
Comment (niccokunzmann) [inline credits.html]: close html in general as I thought.

**Comment by SilentFlame** [comment]
The commented function was that which was getting called when we are clicking on the paint logo link.
I directly directed it to a newly made page ```credits.html. ```.

and about the html page. I'll do it. do I only need to add the header and the closing tag?


Is there any other problem in here.

**Comment by niccokunzmann** [comment]
`onclick="window.location='https://github.com/fossasia/flappy-svg/credits.html';"`
does not work. it should just be "credits.html", so the local copy is served.

File: credits.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+			<h3>These people have contributed to this game: </h3>
+			<ul>
+				<li><a href="http://jugendprogrammiert.weebly.com/">Nicco Kunzmann</a></li>
+				<li><a href="https://github.com/PohSayKeong">Poh Say Keong</a></li>
+				<li><a href="https://github.com/shashank-sharma">Shashank Sharma</a></li>
+				<li><a href="(https://github.com/anniezhou301">Annie Zhou</a></li>
+				<li><a href="https://github.com/harry-7/">Hemanth Kumar Veeranki</a></li>
+				<li><a href="https://github.com/antoniomdk">Antonio Molner Domenech</a></li>
+				<li><a href="https://github.com/toxicdeath32">Siah Bing Z</a></li>
+				<li><a href="(https://github.com/alan-aipe">Alan Aipe</a></li>
```
Comment (niccokunzmann) [inline credits.html]: ( should be removed.

**Comment by niccokunzmann** [comment]
- [ ] If you like, you can make the indentation consistent: open tag -> +1tab close tab -1 tab.
- [ ] remove the brackets in the links
- [ ] remove the commented out function
- [ ] the APK link sould work: Somehow, when I click on Andriod and the on APK, it redirects me to the 404 credits page instead of the APK in the master branch.
- [ ] each open tag should have a corresponding closing tag. Consistent indentation makes this easier to check. Also, some editors such as notepad++ can fold the tags together - nicer to edit.
- [ ] use the link to the local credits.html. It will be downloaded with the app.

This is what I see, currently. I hope the mass does not discourage you. It seems like a lot but most of it can be done fast. If you have any question or you need some idea on how to go on, you can ask me. Also, you can discuss the points with me. It is not like I am your boss :)

**Comment by SilentFlame** [comment]
@niccokunzmann 
Thanks a lot for pointing out the things and guiding me to correct them. I really appreciate your help.
I would like to get a more description regarding the first point you mentioned above.

**Comment by SilentFlame** [comment]
@niccokunzmann  
regarding the APK link. It is working fine in my system. when I click on it this is what happens.No error on my side.
screenshot from 2016-02-04 19 56 57

File: credits.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
+<html>
+<head>
+	<title>Credits</title>
+	<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (niccokunzmann) [inline credits.html]: wrong indentation:
```
+<html>
+<head>
+	<title>Credits</title>
+	<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">

+	</head>
+<body>
```

correct indentation:

```
+<html>
+	<head>
+		<title>Credits</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">

+	</head>
+	<body>
```

As an example: open tag means one more to the right, closing tag means one more to the left.

**Comment by niccokunzmann** [comment]
The APK-link works -> then, no problem. put that as least important and
we can check it if everything else is done.

Am 04.02.2016 um 15:27 schrieb Vinay Singh:

**Comment by SilentFlame** [comment]
I guess First step is done now.
What to proceed in our Second step in this issue?

File: credits.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+					<li><a href="https://github.com/shashank-sharma">Shashank Sharma</a></li>
+					<li><a href="https://github.com/anniezhou301">Annie Zhou</a></li>
+					<li><a href="https://github.com/harry-7/">Hemanth Kumar Veeranki</a></li>
+					<li><a href="https://github.com/antoniomdk">Antonio Molner Domenech</a></li>
+					<li><a href="https://github.com/toxicdeath32">Siah Bing Z</a></li>
+					<li><a href="https://github.com/alan-aipe">Alan Aipe</a></li>
+					<li><a href="https://github.com/SilentFlame">Vinay Singh</a></li>
+				</ul>
+				<h3>Credits To: </h3>
+					<ul>
```
Comment (niccokunzmann) [inline credits.html]: this one would be one more to the left.

File: credits.html [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+	</head>
+<body>
+	<div class="container-fluid">
+		<div class="row">
+			<div class="col-md-4" style="height:100%;background-color:#000">
+			</div>
+			<div class="col-md-4" style="height:100%;background-color:#ffcc7e">
+				<a href="links.html"><img src="back.png" style="position:absolute;width:50px;margin-top:10px;" /></a>
+				
+				<br><br>
```
Comment (niccokunzmann) [inline credits.html]: for tags that do not have a closing tag, in the new HTML style, you would write 

    <br />
    

toindicate that there will not be a closing. Of cause that is understood either way. Just a suggestion.

File: credits.html [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+		<meta charset="utf-8">
+	  	<meta name="viewport" content="width=device-width, initial-scale=1">
+	  	<link rel="stylesheet" href="style/bootstrap/css/bootstrap.min.css">
+	  	<script src="javascript/links.js"></script>
+	  	<script src="javascript/jquery.js"></script>
+	  	<script src="style/bootstrap/js/bootstrap.min.js"></script>
+	  
+	  	<link rel="stylesheet" type="text/css" href="style/links.css"/>
+	</head>
+<body>
```
Comment (niccokunzmann) [inline credits.html]: body indentation would be one to the right because it is in html

File: credits.html [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+				<h1>Contributors</h1>
+				<h3>These people have contributed to this game: </h3>
+				<ul>
+					<li><a href="http://jugendprogrammiert.weebly.com/">Nicco Kunzmann</a></li>
+					<li><a href="https://github.com/PohSayKeong">Poh Say Keong</a></li>
+					<li><a href="https://github.com/shashank-sharma">Shashank Sharma</a></li>
+					<li><a href="https://github.com/anniezhou301">Annie Zhou</a></li>
+					<li><a href="https://github.com/harry-7/">Hemanth Kumar Veeranki</a></li>
+					<li><a href="https://github.com/antoniomdk">Antonio Molner Domenech</a></li>
+					<li><a href="https://github.com/toxicdeath32">Siah Bing Z</a></li>
```
Comment (niccokunzmann) [inline credits.html]: "Siah Bing Ze" - the name of the contributor is misspelled.
https://github.com/fossasia/flappy-svg/blob/master/Contributors.md

**Comment by niccokunzmann** [comment]
Cool. Very good. Things I noticed: 
- [ ] The APK file still redirects to the credits.html (Firefox)
`<div id="x4" class="btn btn-danger" onclick="window.location='credits.html';" style="width:77px;height:77px;padding-left:9px;padding-top:12px;border-radius:135px;"><div style="margin-top:15px;"><b>APK</b></div></div>`
    I get at `TypeError: document.getElementById(...) is null` in this line 13:
    
        	document.getElementById('x3img').style.display="none";
- [ ] a name is misspelled 

Can you check on the both? If you can not reproduce the first one, tell me. You can open the console by clicking right on the page, inspect element and then you may click on console and see the error occurs, when you click on the Android logo to switch to the second view.

**Comment by SilentFlame** [comment]
This commit does most of the expected work.
feedback welcomed :).

**Comment by niccokunzmann** [comment]
Looks good. I will have a closer look, soon and then merge :)

File: credits.html [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+						<li><a href="https://github.com/shashank-sharma">Shashank Sharma</a></li>
+						<li><a href="https://github.com/anniezhou301">Annie Zhou</a></li>
+						<li><a href="https://github.com/harry-7/">Hemanth Kumar Veeranki</a></li>
+						<li><a href="https://github.com/antoniomdk">Antonio Molner Domenech</a></li>
+						<li><a href="https://github.com/toxicdeath32">Siah Bing Ze</a></li>
+						<li><a href="https://github.com/alan-aipe">Alan Aipe</a></li>
+						<li><a href="https://github.com/SilentFlame">Vinay Singh</a></li>
+					</ul>
+					<h3>Credits To: </h3>
+						<ul>
```
Comment (niccokunzmann) [inline credits.html]: This ul is still out of position.

File: javascript/links.js [hunk 11]
```diff
@@ -13,7 +12,7 @@ function android()
 	document.getElementById('x4').innerHTML="<div style='margin-top:15px;'><b>APK</b></div>";
 	document.getElementById('x3img').style.display="none";
 	document.getElementById('x4img').style.display="none";
-}
+	}
 }
 function playstore()
```
Comment (niccokunzmann) [inline javascript/links.js]: this is not the right name for the function.

File: javascript/links.js [hunk 12]
```diff
@@ -13,7 +12,7 @@ function android()
 	document.getElementById('x4').innerHTML="<div style='margin-top:15px;'><b>APK</b></div>";
```
Comment (niccokunzmann) [inline javascript/links.js]: Zou can correct the indentation, if you like.

File: links.html [hunk 13]
```diff
@@ -1,38 +1,35 @@
...
-  <link rel="stylesheet" type="text/css" href="style/links.css"/>
+	<head>
+		<title>Flappy Links</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" href="style/bootstrap/css/bootstrap.min.css">
+		<script src="javascript/links.js"></script>
+		<script src="javascript/jquery.js"></script>
+		<script src="style/bootstrap/js/bootstrap.min.js"></script>
+
```
Comment (niccokunzmann) [inline links.html]: This looks good.

**Comment by niccokunzmann** [comment]
Overall I am satisfied when I look at your contribution. I marked a minor problem: We can not have a function named after what it is not doing.

File: links.html (hunk 13 above)
Comment (SilentFlame) [inline links.html]: I have no idea regarding this. can you please guide me as to what that exactly means.

Comment (niccokunzmann) [inline links.html]: Oh, I just noticed, that this is fine. I just remembered it wrongly! I once saw an error message that jquery could not be loaded but that is out of date now. It loads without warnings and errors. That is something! When you look at many websites, they warn and error a lot.

**Pull request merged**